t/Timer.t

```diff
@@ -32,8 +32,43 @@ ddg_spice_test(
     'timer for 15 mins' => test_spice(@test_args),
     'timer for 77 mins 13 secs' => test_spice(@test_args),
     'timer 2:30' => test_spice(@test_args),
+    'begin timer for 15 mins' => test_spice(@test_args),
+    'run timer for 77 mins 13 secs' => test_spice(@test_args),
+    'start timer 2:30' => test_spice(@test_args),
+    'begin timer with 15 mins' => test_spice(@test_args),
+    'run timer at 77 mins 13 secs' => test_spice(@test_args),
+    'start timer alarm 2:30' => test_spice(@test_args),
+    'begin countdown timer with 15 mins' => test_spice(@test_args),
+    'run timer alarm online at 77 mins 13 secs' => test_spice(@test_args),
+    'start timer alarm for 2:30 online' => test_spice(@test_args),
+    'online start timer alarm for 2:30 online' => test_spice(@test_args),
     'timer 1:20:30' => test_spice(@test_args),
     'countdown for 10 minutes' => test_spice(@test_args),
+    'timer online for ' => test_spice(@test_args),
```

Comment:
I made the trigger "timer for " invalid because it didn't make too much sense to have the word "for" with nothing after it. It doesn't matter too much if it does or doesn't trigger.